svf/include/Util/SVFBugReport.h

```diff
@@ -84,6 +84,7 @@ class GenericBug
 
 public:
     enum BugType {FULLBUFOVERFLOW, PARTIALBUFOVERFLOW, NEVERFREE, PARTIALLEAK, DOUBLEFREE, FILENEVERCLOSE, FILEPARTIALCLOSE};
+    static const std::string BugTypeName[7];
```

Comment:
Why do you want this array? We have the bug type field, right?

Reply:
Can you remove it? I never see it is being used.

Reply:
I think it would be good to make this map as static https://github.com/SVF-tools/SVF/blob/73e01f74d5ff618234e8aacb22f3f4fd3fdaef51/svf/lib/Util/SVFBugReport.cpp#L312